apps/storefront-e2e/src/integration/category-navigation.cy.ts

```diff
@@ -0,0 +1,40 @@
+import { CategoryNavigationFragment } from '../support/page-fragments/category-navigation.fragment';
+import { LandingPage } from '../support/page-objects/landing.page';
+
+describe('Category navigation suite', () => {
+  let landingPage;
+
+  describe('when the landing page is visited', () => {
+    beforeEach(() => {
+      landingPage = new LandingPage();
+      landingPage.visit();
+    });
+
+    it('should render the category navigation in the header ', () => {
+      const navigation = new CategoryNavigationFragment().get();
+      navigation.should('be.visible');
+    });
+
+    it('should have all products link in the category navigation bar', () => {
+      const link = new CategoryNavigationFragment().getLink(0);
+      link.should('contain.text', 'All products');
+    });
+
+    describe('and a navigation item is clicked', () => {
```

Comment:
I'm not quite sure that we have to test links in this way (it might be too expensive, because we pay for each e2e test) 

We should have a unit test that proves that the <a> element with the correct URL/text is rendered and that's enough)